.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ui/inspector"]
+	path = ui/inspector
+	url = https://github.com/freya-fs/freya.ui.inspector.git
```

Comment:
I'm not totally sure what you mean here - the submodule dependency is on a separate repo, which has no dependency back to this - at least, only an implicit one! This submodule provides the content which is embedded in Freya.Inspector to provide a UI. I don't think any kind of cycle is present here, if that's what you mean, but I may just be missing your meaning :)